### ISSUE TITLE
Add IBC genesis docs

### ISSUE THREAD

**Issue by colin-axner** [cosmos/ibc-go#22]
## Summary

Add documentation on how to interact with IBC genesis. There could be potential uses such as creating reserved channels to hubs. 

____

#### For Admin Use

- [x] Not duplicate issue
- [x] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by fedekunze** [comment]
Can you elaborate a bit more on this genesis interaction?

**Comment by colin-axner** [comment]
Yes, so my thinking when I opened this issue is that it is possible to create light clients at genesis reserving certain client identifiers such as: `07-tendermint-0` to `07-tendermint-10` or something. Then you can set the starting client identifier index at `100`. This allows the chains validators to say which first 10 clients can be trusted at the same level of trust we trust the validator set. 

It is kinda difficult and requires coordination, even more so with trying to reserve connections/channels. 

At the very least we should have some explanation about when any IBC genesis state should be included, which is probably only on genesis state restarts. But maybe we should talk about `historical_info` and transfer enablement, and allowed clients, etc
